packages/sdks-tests/src/tests/duplicate-attributes.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { expect } from '@playwright/test';
+import { test } from './helpers.js';
+
+test.describe('Duplicate Attributes', () => {
+  test('wrapped block has no duplicate attributes', async ({ page, packageName }) => {
+    await page.goto('/duplicate-attributes');
```

Comment:
you're going to want to `test.skip(packageName === 'react-native')` to ignore the test in react native, because it doesn't have HTML attributes in the same way.

Reply:
Thanks! Got it!